### DISCUSSION TITLE
Reality 协议现在是否已经会被 GFW 识别到了？

### DISCUSSION THREAD

**Discussion by EsunR** [XTLS/Xray-core#3540] [General]
从7月15日开始，我的一个使用 reality 协议的节点开始出现无法连通的情况，更换一个端口后恢复正常，无法连通的端口在大约10小时后又能重新使用了。

今天，也就是7月16日上午，我的另外一个节点也出现了无法连通的情况（也是端口被封），截至目前仍然无法访问。

**Comment by RPRX** [comment]
标题有点难评，密码学安全性上，REALITY 在流行代理协议中是最强的，强于 TLS，至于会不会被中国 GFW 封，目前没有收到大规模报告。

如果没有大规模报告，表明中国 GFW 尚未开始针对性封锁该协议。即使是伊朗 GFW 也是表现出乱封，封了很多正常的网站。

**Reply by EsunR** [reply]
抱歉，调整了一下标题。目前我这边遇到的情况是两台不服务商 VPS 创建的节点都出现了端口封禁的问题，电信和联通的网络都测试了一下无法使用，应该可以排除是运营商的问题，之前使用没有出现这种情况

**Reply by RPRX** [reply]
@EsunR REALITY 有很多种用法你要说出你是怎么用的，~~想搞个 discussion 模板了~~，比如“VLESS-Vision-REALITY 偷白名单域名”和“VLESS-noflow-REALITY 偷自己”天差地别，后者肯定不抗封

此外 UDP 类代理昨天开始被大规模封锁，最近好像是特殊时期，特殊时期多封一些很正常，但目前总体上 REALITY 存活状况良好

**Reply by WilliamSoldier** [reply]
CMCC 可以用

**Reply by violetice** [reply]
反正我的一个香港ip已经寄了，VLESS-HTTP2-REALITY

**Comment by xingpingcn** [comment]
我的REALITY 和hy2稳定性差不多，用不了的时候大家一起用不了，只能切手机卡。不过基于udp的hy2还是要快许多

**Comment by yzh118** [comment]
SNI一定要设置为“www.gov.cn”一定会霉逝的

**Reply by drgtdrgtgsd** [reply]
自投罗网是吧

**Reply by CloudSVC** [reply]
自投罗网这一块
